src/annotator/guest.ts

```diff
@@ -458,6 +458,31 @@ export class Guest extends TinyEmitter implements Annotator, Destroyable {
     this._hostRPC.connect(hostPort);
   }
 
+  /**
+   * Scroll an anchor into view and notify the host page.
+   *
+   * Returns a promise that resolves when scrolling has completed (see
+   * {@link Integration.scrollToAnchor})..
+   */
+  async _scrollToAnchor(anchor: Anchor) {
```

Comment:
Is this method intended to be private?

```suggestion
  private async _scrollToAnchor(anchor: Anchor) {
```